jdbc/src/test/java/org/apache/iotdb/jdbc/IoTDBQueryResultSetTest.java

```diff
@@ -240,7 +241,7 @@ public void testQuery() throws Exception {
     }
 
     // The client issues 2 fetchResultReq in total. The last fetchResultReq get empty resultSet.
-    verify(fetchResultsResp, times(2)).getStatus();
+    verify(fetchResultsResp, times(1)).getStatus();
```

Comment:
I think the comment also needs to be updated

Reply:
updated